package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.5.0",
   "description": "ESLint plugin to prevent use of extended native objects",
   "scripts": {
-    "compile": "babel src --out-dir rules",
+    "compile": "babel src --source-maps --out-dir rules",
```

Comment:
What's the value of of creating the source maps? Does this help consumers of this plugin? Or is it to help with local development?

Reply:
If there is an error, the user (if a technical enough user) will know the exact line in source to find the problem, and if not a technical user, if they report the trace to the project, it will make it faster for us to find the problem.

It also helps with testing coverage so long as your tests are pointing to `rules` rather than `src`--ensuring that if there are any problems (now with 100% coverage, that would only be for regressions) the proper line numbers would be shown. (While you could point to `src` in the test file, an advantage of keeping it as is is that it can fail if there are build problems.)

Reply:
WHOA! I didn't know Node added support to use source maps in stack traces. That is amazing. Thank you for sharing! I've been out of the loop on Node stuff. :)